docs/source/mayavi/installation.rst

```diff
@@ -181,44 +181,51 @@ Installing from `Conda`
 
 Conda is an open source package management environment management system for installing multiple versions of software packages and their dependencies. Conda is included in Anaconda and Miniconda.
 
-Miniconda is a small "bootstrap" version that includes conda, python, and the packages they depend on. You can install miniconda from the following link depending on your operating system.
+Miniconda is a small "bootstrap" version that includes conda, python, and the packages they depend on. Depending on your operating system, you can download the `MinicondaInstaller <https://conda.io/miniconda.html>`_
 
-For python 3.6: 
---------------
 
-`Windows64bit <https://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86_64.exe>`_
+After installing conda you need to run the following from command line:: 
 
-`Windows32bit <https://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86.exe>`_
+  $ conda create -n pyconda python=3.5 pyqt=4
+  $ source activate pyconda
+  $ conda install -c menpo mayavi
 
-`MacOS <https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh>`_
+You should be all set with this.
 
-`Linux64bit <https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh>`_
 
-`Linux32bit <https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86.sh>`_
+.. Installing with conda-forge 
 
+Installing with `Conda-forge`
+-----------------------------
 
-For python 2.7: 
---------------
+Conda-forege is a community-led conda channel of installable packages. For more information on conda-forge, you can vist their their website_.
 
-`Windows64bit <https://repo.continuum.io/miniconda/Miniconda2-latest-Windows-x86_64.exe>`_
+.. _website: https://conda-forge.org
 
-`Windows32bit <https://repo.continuum.io/miniconda/Miniconda2-latest-Windows-x86.exe>`_
+Follow these steps to install using conda-forge: 
 
-`MacOS <https://repo.continuum.io/miniconda/Miniconda2-latest-MacOSX-x86_64.sh>`_
+First, clone the environment or build one from scratch::
 
-`Linux64bit <https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh>`_
+	$ conda create --name pyforge --clone root
+ 	or
+ 	$ conda create --name pyforge python=2.7	
```

Comment:
Why only 2.7?  I think we should not suggest 2.7 unless people need it.  Just use 3.x the default for these.